### ISSUE TITLE
Disable Visual Editor

### ISSUE THREAD

**Issue by Testbusters** [QuizandSurveyMaster/quiz_master_next#665]
Visual editor sometimes is automatically changing some part of the code. That's why we disabled it on our website, but this seems to break the plugin.
Is there a way to make it work or set "Text" as default instead of visual editor?

## General description
I don't know if it's actually a bug or a feature request, because I don't think the plugin should break if you disable visual editor, but I know most of wordpress users are using visual editor.

## Expected behavior
Quixmaster is not breaking if I disable visual editor, but simply select "Text" instead of "Visual" (this is also the normal behaviour for pages and articles)

## Actual behavior
The plugin breaks

## Steps to reproduce the behavior
disable the Visual editor on your profile, create a survey, add a question and try to edit the question, the editor is not going to open.

**Comment by fpcorso** [comment]
Hmmm very interesting @Testbusters. If I disable Visual Editor and try to open a survey with pre-existing questions, the questions never load as well.

I looked through the JS console and found:

`Uncaught TypeError: Cannot read property 'translate' of undefined`

I looked through the file and it appears to be coming from the wp.editor.initialize function. I would have thought the WordPress function for initializing would account for users with the visual editor disabled but it appears to not be the case.

Let me do some research to see if there is a different way to handle this use case.
